.eslintrc.json

```diff
@@ -9,5 +9,12 @@
     "jest/globals": true,
     "node": true
   },
-  "plugins": ["jest"]
+  "rules": { "no-console": "error" },
+  "plugins": ["jest"],
+  "overrides": [
```

Comment:
Could we have overrides immediately after rules, just so it's really clear they are linked.

Reply:
Yep, done!